### DISCUSSION TITLE
Fire Island!

### DISCUSSION THREAD

**Discussion by diasfigor** [sunflower-land/sunflower-land#2360] [Ideas]
Hello everyone @0xsunflowerstudios !

I am suggesting a new content for our game, a **Fire Island**!

Below I will explain everything about it:

**1. Lore**
**2. Mechanics**

**_1. Lore_**

Is the war over?

Long ago we were driven out of our home by the Skeleton King and his skeleton order. Our homes were destroyed and we were forced to get into a sailboat and sail out to sea in search of a new there. Thanks to the Sunflower Gods we found an island with enough resources to grow and feed ourselves, and with a lot of work we managed to turn this island into a great place to live.

Days went by and more and more technologies were discovered, making our life more pleasant and productive. We relearned how to harvest our old crops and learned how to plant fruit trees and harvest their fruit. New foods and drinks have been developed to nourish us and make us grow stronger and ready for new adventures.

Behold, the answer to the initial question was given: No!

On one of our ocean voyages we saw a gray cloud in the distance, the sea started to become sluggish, the wind began to sway the boat, and suddenly a frightening roar. In the distance we could see giant flaming balls falling into the sea, raising the height of the waves making our small boat be thrown from one side to the other almost sinking us, but with the dexterity of our Bumpkins managed to overcome the waves.
We had no way back, our boat was broken because of the big waves and pieces of what seemed to be volcanic lava that fell from the sky.
Our fear was great but our curiosity to discover what was beyond the gray clouds was much greater. We continued in the rough and stormy sea towards the gray and dense wall of clouds that was in front of us. 

With much sacrifice and strength in the arms we managed to cross the barrier and reach beyond the clouds. 
And there was what we feared most, the reason for our war, the ordeal of evil skeletons. A doubt hung over our heads: Is the Skeleton King near? Our fear became even greater, but in our chests grew an enormous desire to answer this question.

There was no way back, when we reached the island they saw us, but we managed to hide close to the rock. We waited a few hours for the skeletons to leave.

Curiosity struck again and we went very slowly following the skeletons, when in the distance we saw the Skeleton King on top of a large Volcano about to erupt. 

We heard in the distance the voice of the Skeleton King shouting and shaking the volcano.

The Air was dense and the temperature of the island was very hot and difficult for us Bumpkins to stay for long, so with few resources we created a special outfit that helped us stay longer on the island.

Unfortunately we had no other way of getting off the island, if not to finally have to face the Skeleton King, pack our boat and return to our home.

**And now... will we win?**

**_2. Mechanics._**

Sunflower Team, I had the idea to create an island with the name of: **_Fire Island_** where we will have an active volcano called: **_SunFireFlower_** that from time to time (once a month) will gush lava with some 'prize' for the player, it would be rare NFT with BUFF, decorative or clothes for the Bumpkin. 

**How the island will work:**
 - The player will have to be level 40 or higher to enter the island;
 - Once a month, the island will appear and remain for a set amount of time for the player to enter and search for their reward amidst the volcanic lava;
 - In order for the player to enter the island, he will have to make a specific outfit for his Bumpkin to be protected against the intense heat of the lava (consumable outfit), otherwise he will not be allowed to enter;
 - When the player enters the island, a 45 minute countdown will start for the player to look for the item that is hidden on the island (hidden indeed). The prize will not be in a chest, as usual. (There will be an area similar to the Treasure Island area, for the player to click with the mouse looking for the item)
- If the player does not find the item in the given time, he will be expelled and will return to his original island, will lose the clothes created and will be able to create it again after 25 days from the first creation (this is to give time for the island to appear again);
 - Items on the island are not cumulative;
 
**Feed the Volcano:**
 - Once a month the island will dive from the sea asking for food for the volcano that is about to erupt, it could be cakes, fruits, etc;
 - In this appearance, the volcano will not be gushing lava yet, at this stage there will be no need for the clothing to search for the item;
 - If the player does not feed the volcano, it will erupt, sinking the island and may not enter the island for 1 to 3 cycles (1 to 3 months);
 - In order to re-enter the island, the player will have to pay a fee to calm the volcano and clean the island, which can be in SFL (20, 40, 60, 80 or 100 SFL);
 - By paying this fee the player can, depending on the cycle, re-enter the island when it reappears.

*The idea of NFT drops would not be one for all, but there would be several, so that players have different items, so that if they do not like them, they can sell them, thus moving the game's economy in relation to the rarity of the NFT's.*

**_Contribution SubZero and Teka._**

**Comment by IngridFirmino** [comment]
Sunflower, the best game of the world! Little fireeeee

**Comment by Celinhotv** [comment]
Nice idea!!

I think it's good to add new objectives for the High lvl players to spend resources and time.
